templates/js/README.md

```diff
@@ -0,0 +1,13 @@
+# %APPNAME%
+
+## Getting Started
```

Comment:
Let's replace this README with the one from quickstart

https://github.com/nearbuilders/quickstart